addons/csi/openstack/controllerplugin.yaml

```diff
@@ -169,7 +182,11 @@ spec:
         - name: socket-dir
           emptyDir:
         - name: cloud-config
+          configMap:
+            name: cloud-config
+        - name: kubeconfig
           secret:
-            secretName: cloud-config
+            secretName: internal-admin-kubeconfig
```

Comment:
the CSI sidecar containers are capable of being pointed to a different cluster. Should we create designated kubeconfig just for this CSI plugin or is it ok for it to use the `internal-admin-kubeconfig'?